WordPress/Classes/Extensions/Post+BloggingPrompts.swift

```diff
@@ -1,76 +1,11 @@
-
 extension Post {
 
-    func prepareForPrompt(_ prompt: Prompt?) {
+    func prepareForPrompt(_ prompt: BloggingPrompt?) {
         guard let prompt = prompt else {
             return
         }
-        postTitle = prompt.postTitle
-        let pullquoteBlock = getPullquoteBlock(title: prompt.promptText,
-                                               promptUrl: prompt.promptUrl?.absoluteString,
-                                               answerUrl: prompt.answerUrl?.absoluteString,
-                                               answerCount: prompt.answerCount)
-        content = pullquoteBlock + Strings.emptyParagraph
+        postTitle = prompt.title
```

Comment:
Just to note, I added a task on [this issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/18375) to create a custom local title since the remote prompt.title is not user friendly.